data/lang/equipment-core/en.json

```diff
@@ -287,46 +327,202 @@
+    "description": "Name for a single-missile external launcher. 'Cnida' is a model name.",
     "message": "Cnida-101 Missile Rail"
   },
+  "MISSILE_RAIL_S1_DESCRIPTION": {
+    "description": "Equipment description",
+    "message": "External launcher for a single small missile."
+  },
+  "MISSILE_RAIL_S1_FLAVOURTEXT": {
+    "description": "Equipment description; 'Hydri Defense' is a manufacturer name.",
+    "message": "Hydri Defense's entry point rail, this is a quality piece of equipment for all S1 missiles adhering to ISAAEEE standards."
+  },
+
   "MISSILE_RAIL_S2": {
-    "description": "Name for a single-missile external launcher. 'Cnida' is a brand name.",
+    "description": "Name for a single-missile external launcher. 'Cnida' is a model name.",
     "message": "Cnida-102 Missile Rail"
   },
+  "MISSILE_RAIL_S2_DESCRIPTION": {
+    "description": "Equipment description",
+    "message": "External launcher for a single medium missile."
+  },
+  "MISSILE_RAIL_S2_FLAVOURTEXT": {
+    "description": "Equipment description; 'Hydri Defense' is a manufacturer name.",
+    "message": "A quality missile rail from Hydri Defense, this provides a secure mounting point for S2 class missiles."
+  },
+
   "MISSILE_RAIL_S3": {
-    "description": "Name for a single-missile external launcher. 'Cnida' is a brand name.",
+    "description": "Name for a single-missile external launcher. 'Cnida' is a model name.",
     "message": "Cnida-103 Missile Rail"
   },
+  "MISSILE_RAIL_S3_DESCRIPTION": {
+    "description": "Equipment description",
+    "message": "External launcher for a single large missile."
+  },
+  "MISSILE_RAIL_S3_FLAVOURTEXT": {
+    "description": "Equipment description; 'Hydri Defense' is a manufacturer name.",
+    "message": "A quality missile rail from Hydri Defense, this provides a secure mounting point for S3 class missiles."
+  },
+
+  "MISSILE_RAIL_S4": {
+    "description": "Name for a single-missile external launcher. 'Cnida' is a model name.",
+    "message": "Cnida-104 Missile Rail"
+  },
+  "MISSILE_RAIL_S4_DESCRIPTION": {
+    "description": "Equipment description",
+    "message": "External launcher for a single huge missile."
+  },
+  "MISSILE_RAIL_S4_FLAVOURTEXT": {
+    "description": "Equipment description; 'Hydri Defense' is a manufacturer name.",
+    "message": "A quality missile rail from Hydri Defense, this provides a secure mounting point for S4 class missiles."
+  },
+
   "MISSILE_SMART": {
     "description": "",
     "message": "Smart Missile"
   },
+  "MISSILE_SMART_DESCRIPTION": {
+    "description": "Equipment description",
+    "message": "A guided missile somewhat resistant to ECM."
+  },
+  "MISSILE_SMART_FLAVOURTEXT": {
+    "description": "Equipment description",
+    "message": "A smart rocket capable of tracking down your target using an advanced advanced sensors with an integrated AI module. Make sure not to call it HAL and we're sure it will home in on your intended target, and not you.."
```

Comment:
Duplicated word here - "advanced advanced sensors".

Reply:
Regression introduced in 1a2992f687dc5d7cbdf2b3608c3149665e41eeca; added a fixup commit to fix this. Thanks!